backend/macros/src/config.rs

```diff
@@ -119,6 +271,13 @@ fn gen_toml(input: &Input) -> String {
 }
 
 
+fn to_camel_case(ident: &Ident) -> Ident {
```

Comment:
This only works for single words, though, right? o.O

Reply:
Indeed, adding something like this to the config macro invocation:

```rust
some_thing: {
    /// A foo
    foo: u16 = 42,
},
```

generates two warnings:

```
warning: type `Some_thing` should have an upper camel case name
  --> server/src/config.rs:23:5
   |
23 |     some_thing: {
   |     ^^^^^^^^^^ help: convert the identifier to upper camel case: `SomeThing`
   |
   = note: `#[warn(non_camel_case_types)]` on by default

warning: type `Some_thing` should have an upper camel case name
  --> server/src/config.rs:23:5
   |
23 |     some_thing: {
   |     ^^^^^^^^^^ help: convert the identifier to upper camel case: `SomeThing`

warning: 2 warnings emitted
```